scheduler/scheduler-server/build.gradle

```diff
@@ -31,9 +31,8 @@ dependencies {
     compile project(':scheduler:scheduler-node')
     compile project(':rm:rm-server')
     compile project(':rm:rm-client')
-
-
-
+    //compile project(':rest:rest-server')
```

Comment:
The commented out line should be removed.

Reply:
Fixed